app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,7 @@
+class GuestsController < ApplicationController
+
+  def show
+    @guest = Guest.includes(rooms: :hotel).find(params[:id])
+  end
```

Comment:
I see that you're using `.includes` here, what exactly is it doing? Normally you would want to just use a `.find` in your `show` action.

```ruby
def show
  @guest = Guest.find(params[:id])
end
```